src/components/Reports/MPGAIncomeExpensesReport/Helper/getMonthCount.ts

```diff
@@ -0,0 +1,14 @@
+export function getMonthCount(
```

Comment:
[Reuse] Similar month parsing logic exists in getMonthInfo.ts. Evidence: Both files split monthYear by space to extract year. Impact: Code duplication. Suggestion: Extract shared utility function. [Category: Reuse]